src/Stratis.Bitcoin.Features.RPC/Controllers/FullNodeController.cs

```diff
@@ -175,6 +177,44 @@ public BlockHeaderModel GetBlockHeader(string hash, bool isJsonFormat = true)
             return model;
         }
 
+        /// <summary>
+        /// Return information about <address>.
+        /// </summary>
+        /// <param name="address">BitcoinAddress to validate.</param>
+        /// <returns>information about <address>. {isvalid}</returns>
+        [ActionName("validateaddress")]
+        [ActionDescription("Returns information about a bitcoin address")]
+        public JObject ValidateAddress(string address)
+        {
+            if(address == null)
```

Comment:
Maybe also add a check for string.Empty? I think you can use string.IsNullOrEmpty